### ISSUE TITLE
HTTPFound error

### ISSUE THREAD

**Issue by dderiso** [falconry/falcon#677]
falcon.HTTPFound('http://www.google.com/')
throws
AttributeError: 'module' object has no attribute 'HTTPFound'

**Comment by dderiso** [comment]
this workaround solves it
```
def on_get(self, req, res):
            res.status = falcon.HTTP_302
            res.location = 'http://www.google.com/'
```

**Comment by bendemaree** [comment]
Not all HTTP status "shortcuts" are available in the top-level namespace; `HTTPFound` is [not among them](https://github.com/falconry/falcon/blob/master/falcon/errors.py).

You could try:
```
from falcon.http_status import HTTPStatus
HTTPStatus(falcon.HTTP_302, headers={"Location": "http://www.google.com/"})
```

On `master`, `HTTPStatus` is [in the top-level namespace](https://github.com/falconry/falcon/commit/e0e391d422cbe45a3ef97f8f960a2e40a0a05c19):

```
falcon.HTTPStatus(falcon.HTTP_302, headers={"Location": "http://www.google.com/"})
```

**Comment by kgriffs** [comment]
falcon.HTTPFound is available as of version 1.0.0 of the framework
